src/HH4b/processors/bbbbSkimmer.py

```diff
@@ -807,9 +810,17 @@ def process(self, events: ak.Array):
         }
 
         if self._region == "signal":
-            for jshift in ["", "JMS_down", "JMS_up", "JMR_down", "JMR_up"] + list(
-                self.jecs.values()
-            ):
+            for jshift in [
+                "",
+                "JMS_down",
```

Comment:
@cmantill We need to add the current JMR/JMS scale factors to the processor if we want to use these right?

Reply:
As we discussed offline @cmantill I think the current approach is fine, unless you want to update it with the new numbers from the Loukas method.